### ISSUE TITLE
[BUG]error running the jail command: exit status 1

### ISSUE THREAD

**Issue by hyunkim13** [rancher/rancher#43311]
**Rancher Server Setup**
- Rancher version: v2.6.3
- Installation option (Docker install/Helm Chart):
   - Docker Install 

[Build and run environment]
**ubuntu 20.04 / docker version 20.10.10**
**ubuntu 18.04 / docker 20.10.9**

[Build command] 
**make**
[Run command]
**docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher:dev**

What I edited
script/ci
"./validate" and "./test" remove

package/Dockerfile
153 Line ARG VERSION=dev => ARG VERSION=v2.6.3

I have been building and using rancher/server myself since two years ago.
I was using it well until the beginning of this year, but recently the container is not running.
The build is successful, but the container continues to restart when running docker run.
If you check with docker log, the [FATAL] error running the jail command: exit status 1 log is repeated.
![image](https://github.com/rancher/rancher/assets/89969109/4c3d86d6-5023-4f76-92c3-7136087e4ed8)

I tried running it on various ubuntu versions and docker versions, but it doesn't work.
The previous version, v2.4.3, can be built and run.
However, the version used, v2.6.3, does not run.

![image](https://github.com/rancher/rancher/assets/89969109/47c2579a-e2e4-48d2-bd79-311df9b6049f)
![image](https://github.com/rancher/rancher/assets/89969109/26a4b1ec-8a01-48b9-beee-6c2464957fa2)

**Comment by Dlimingliang** [comment]
Rancher images currently being built no longer run due to the missing package openssh-clients. jail.sh [uses the ssh binary](https://github.com/rancher/rancher/blob/release/v2.8/package/jailer.sh#L76) when running and causes Rancher to exit if it does not exist.

**Comment by Dlimingliang** [comment]
Rancher images currently being built no longer run due to the missing package openssh-clients. jail.sh when running and causes Rancher to exit if it does not exist.
I added openssh-clients., it works for me。It's a Dockerfile file under the package directory.
`RUN zypper -n install --no-recommends git-core curl ca-certificates unzip xz gzip sed tar shadow gawk vim-small netcat-openbsd mkisofs openssh-clients && \
    zypper -n clean -a && rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/* && \
    useradd rancher && \
    mkdir -p /var/lib/rancher /var/lib/cattle /opt/jail /opt/drivers/management-state/bin && \
    chown -R rancher /var/lib/rancher /var/lib/cattle /usr/local/bin`
I think you might have the same issue. You can try it and see。

**Comment by herlon214** [comment]
I can see they added the `openssh-clients` on [v2.7.9](https://github.com/rancher/rancher/releases/tag/v2.7.9) check the [commit](https://github.com/rancher/rancher/commit/0c020f0c6846ddfe1c66e2a9f8ac1ebef3b171c6).

**Comment by hyunkim13** [comment]
I found the problem with that error.
I checked the log by adding the --trace option when running docker.
ex)docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.3 --trace

The error occurred because there was no ssh in the base image, suse/sle15:15.3.
I built rancher using a custom image with the /usr/bin/ssh file added.
Thank you to the two people who helped.
